test/formulator_test.exs

```diff
@@ -1,8 +1,37 @@
 defmodule FormulatorTest do
   use ExUnit.Case
   doctest Formulator
+  alias Phoenix.HTML.Form
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  describe "input - label" do
+    test "a label is created with the field name" do
+      [{:safe, label} | _] = %Form{data: %{name: ""}}
+      |> Formulator.input(:name)
+
+      assert label |> to_string == "<label for=\"_name\">Name</label>"
+    end
+
+    test "passing the label text option allows for overring the label" do
+      [{:safe, label} | _] = %Form{data: %{name: ""}}
+      |> Formulator.input(:name, label_text: "Customer Name")
+
+      assert label |> to_string == "<label for=\"_name\">Customer Name</label>"
+    end
+  end
+
+  describe "input - form input" do
+    test "passing classes puts then on the input" do
+      [_, {:safe, input} | _] = %Form{data: %{name: ""}}
+      |> Formulator.input(:name, class: "customer_name")
+
+      assert Regex.match?(~r/class=\"customer_name\ "/, input)
```

Comment:
I think using `=~` is a bit more clear in cases like these 

```elixir
assert input =~ ~s(class="customer_name")
```

I also used sigil_s so that it reads a bit better. What do you think?